bin/git-theta

```diff
@@ -39,49 +44,64 @@ def parse_args():
     )
     track_parser.set_defaults(func=track)
 
-    args = parser.parse_args()
+    add_parser = subparsers.add_parser(
+        "add", help="add command used to stage files."
+    )
+    add_parser.add_argument("--update-type", choices=["dense", "sparse"], help="Type of update being applied")
+    add_parser.set_defaults(func=add)
+
+    args = parser.parse_known_args()
     return args
 
 
-def paths_and_parameters(model_dict, root):
-    """Convert the keys of the model dict to parameters based starting at root."""
-    return {os.path.join(root, *k): v for k, v in utils.flatten(model_dict).items()}
+def post_commit(args):
+    repo = git_utils.get_git_repo()
+    theta_commits = theta.ThetaCommits(repo)
+
+    gitattributes_file = git_utils.get_gitattributes_file(repo)
+    patterns = git_utils.get_gitattributes_tracked_patterns(gitattributes_file)
+
+    oids = set()
+    commit = repo.commit('HEAD')
+    for obj in commit.tree.traverse():
```

Comment:
This seems to iterate over all the objects in the git repo. Is there an issue with using `commit.stats.files` to just check the files that were changed in this commit?

Reply:
Good idea. Fixed